apps/language_server/lib/language_server/server.ex

```diff
@@ -208,6 +208,7 @@ defmodule ElixirLS.LanguageServer.Server do
 
     if reason == :normal do
       WorkspaceSymbols.notify_build_complete()
+      Formatting.build_cache(state.project_dir)
```

Comment:
OK so this solves the issue with formatting not being able to pull in deps initially. By re-building the the cache after every build we trivially catch updates to deps or other changes that effect formatting.